lib/SILOptimizer/Mandatory/TFLowerGraph.cpp

```diff
@@ -143,10 +143,13 @@ namespace {
     SmallVector<std::pair<SILArgument*, TF_Output>, 4> outputs;
 
     /// If this graph has any side-effecting operations, this is the most
-    /// recently emitted operation that had side effects.  Otherwise, it is
-    /// null.
+    /// recently emitted operation that had side effects, and that operation
```

Comment:
"that operation" -> "this operation", since it was "this" in the previous line.

Reply:
Done